### DISCUSSION TITLE
share .github/release.yml on .github org repo

### DISCUSSION THREAD

**Discussion by jetersen** [community/community#7926] [Other Feature Feedback, Questions, & Ideas] [answered]
Would be great if automatic release notes would read the .github/release.yml from a .github repo like so many other github features do already support.

I do not want yet another file that creates configuration debt 😅 Yaa for reusable workflows

**Comment by MylesBorins** [comment, marked as answer]
This is a great idea @jetersen. I've made a feature request, but to set expectations it might take us a while to prioritize this work

**Reply by marcomontalbano** [reply]
@MylesBorins can you link here the feature request? So that we can track the status. Thanks!!

**Reply by marcomontalbano** [reply]
Hi @MylesBorins do you have any update?

**Reply by adoreleon** [reply]
Hi @MylesBorins ! Do you have any updates on this? That would help a lot and its been 2 years since this question was created 😆

**Comment by dhirschfeld** [comment]
> I've made a feature request

Is that available somewhere open so we can track any progress, or is this the best place to get any updates?
